### PR TITLE
[00004-0] Implementing CAN for transmitter. Separating transmitter and receiver target builds

### PR DESCRIPTION
**Pull request by 0x2A9**

### CODE REVIEW

File: src/drivers/can.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    if (CAN_TransmitStatus(CAN1, mb) == CAN_TxStatus_Failed) {
+        return 1;
+    }
+
+    while (CAN_TransmitStatus(CAN1, mb) == CAN_TxStatus_Pending);
+
+    return 0;
+}
```
Comment (gilsllendo) [inline src/drivers/can.c]: Invalid formatting

File: src/rx/main.c [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
+            pca9685_set_pwm(3, 0, counter);
+
+            counter -= 1;
+        }
+
+        GPIO_SetBits(GPIOE, GPIO_Pin_8 | GPIO_Pin_11);
+    }
+
+    return 0;
+}
```
Comment (gilsllendo) [inline src/rx/main.c]: Invalid formatting

File: src/tx/main.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+int main(void)
+{
+    gpio_init();
+
+    if (can_init() == CAN_InitStatus_Success) {
+        GPIO_SetBits(GPIOE, GPIO_Pin_8);
+    }
+
+	msg.StdId = 0;
```
Comment (gilsllendo) [inline src/tx/main.c]: Invalid formatting

**Review by gilsllendo — CHANGES_REQUESTED** [review]

File: src/drivers/can.c (hunk 1 above)
Comment (0x2A9) [reply src/drivers/can.c]: Done.

File: src/rx/main.c (hunk 2 above)
Comment (0x2A9) [reply src/rx/main.c]: Done.

File: src/tx/main.c (hunk 3 above)
Comment (0x2A9) [reply src/tx/main.c]: Done.

**Review by gilsllendo — APPROVED** [review]

**Pull request merged**